packages/ibm-products-web-components/src/components/options-tile/options-tile.ts

```diff
@@ -0,0 +1,137 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html, nothing } from 'lit';
+import { property } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { prefix } from '../../globals/settings';
+import HostListenerMixin from '@carbon/web-components/es/globals/mixins/host-listener.js';
+import styles from './options-tile.scss?lit';
+import { carbonElement as customElement } from '@carbon/web-components/es/globals/decorators/carbon-element.js';
+import ChevronDown20 from '@carbon/web-components/es/icons/chevron--down/20';
+
+const blockClass = `${prefix}--options-tile`;
+
+/**
+ * OptionsTile.
+ *
+ * @element c4p-options-tile
+ * */
+
+@customElement(`${prefix}-options-tile`)
+class CDSOptionsTile extends HostListenerMixin(LitElement) {
+  /**
+   * If `true` the body of the component is shown
+   */
+  @property({ type: Boolean })
```

Comment:
```suggestion
  @property({ type: Boolean, reflect: true })
```

I think we've been adding reflect true to properties